py/src/nupic/encoders/coordinate_encoder.py

```diff
@@ -0,0 +1,35 @@
+
+import nupic.bindings.encoders
+CoordinateEncoder           = nupic.bindings.encoders.CoordinateEncoder
+CoordinateEncoderParameters = nupic.bindings.encoders.CoordinateEncoderParameters
```

Comment:
why do you use this construction? And not `from nupic.bindings... import Coord, CoordParams` ?

Reply:
what must be set/imported so I can use the standard `from htm.bindings.encoders import Coord` ? Currently that is failing in the py tests